software/base/pom.xml

```diff
@@ -116,7 +116,6 @@
             <groupId>org.apache.brooklyn</groupId>
             <artifactId>brooklyn-locations-jclouds</artifactId>
             <version>${project.version}</version>
-            <scope>test</scope>
```

Comment:
Why remove it from here, but leave all following dependencies the same. If you need a (non-test) dependency on `brooklyn-locations-jclouds` add a new one instead.